ci/cpu/build.sh

```diff
@@ -49,17 +53,28 @@ conda config --set ssl_verify False
 # BUILD - Conda package builds (conda deps: libcupatial <- cuspatial)
 ##########################################################################################
 
-logger "Build conda pkg for libcuspatial..."
-cd $WORKSPACE
-source ci/cpu/libcuspatial/build_libcuspatial.sh
+gpuci_logger "Build conda pkg for libcuspatial"
+if [ "$BUILD_LIBCUSPATIAL" == '1' ]; then
+  if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
+    conda build conda/recipes/libcuspatial
+  else
+    conda build --dirty --no-remove-work-dir conda/recipes/libcuspatial
+  fi
+fi
 
-logger "Build conda pkg for cuspatial..."
-source ci/cpu/cuspatial/build_cuspatial.sh
+gpuci_logger "Build conda pkg for cuspatial"
+if [ "$BUILD_CUSPATIAL" == '1' ]; then
+  if [[ -z "$PROJECT_FLASH" || "$PROJECT_FLASH" == "0" ]]; then
+    conda build conda/recipes/cuspatial
+  else
+    conda build --dirty --no-remove-work-dir conda/recipes/libcuspatial
```

Comment:
This should be cuspatial. And should include the local artifact channel (https://github.com/rapidsai/rmm/commit/14e144696b1074a8f2bf0b064bcd61d850d48e79)